pkg/util/net/ip.go

```diff
@@ -33,3 +37,17 @@ func IsIPv4Address(ip string) bool {
 	ipa, _ := netip.ParseAddr(ip)
 	return ipa.Is4()
 }
+
+func IPsSplitV4V6(ips []string) (ipv4 []string, ipv6 []string) {
```

Comment:
Also a nit, but I'm not sure if it's worth changing this to return strings instead of net.IP. We end up re-parsing it when answering DNS queries.

Reply:
I think you are right, updated